README.md

```diff
@@ -83,10 +83,27 @@ components in the Pulsar ecosystem, including connectors, adapters, and other la
 
 - [Pulsar CI](https://github.com/apache/pulsar-test-infra)
 
+## Pulsar Runtime Java Version Recommendation
+
+pulsar ver 2.11  >=
+
+| Pulsar Components | Java Version  |
+| ----------------- |:-------------:|
+| Broker            | 17            |
+| CLI               | 17            |
+| Java Client       | 8 or 11 or 17 |
+
+pulsar ver 2.11 <
+
+| Pulsar Components | Java Version  |
+| ----------------- |:-------------:|
+| All               | 8 or 11       |
```

Comment:
For Broker, the recommended runtime environment has been Java 11 since Pulsar 2.8.0 release.